pkgs/tools/security/gnupg/20.nix

```diff
@@ -3,13 +3,13 @@
 
 # Each of the dependencies below are optional.
 # Gnupg can be built without them at the cost of reduced functionality.
-, pinentry ? null, x11Support ? true
+, pinentry ? null, guiSupport ? true
```

Comment:
@veprbl This all used to make sense before gnupg switched to their brand new model centered around gpg-agent. You are right in that, as far as I can tell, nowadays it is hard to run gnupg without pinentry (although not impossible, there is an option to gpg-agent called `allow-loopback-pinentry`, and while I’m not sure how exactly it works, it seems that it instructs gpg to ask for the password on the terminal).

I still think that this `guiSupport` option might be useful as a quick shortcut, it’s just that instead of disabling pinentry completely it should adjust the flags of the default pinentry accordingly. But anyway, this is an issue separate from this one.